qa/automation/CareerPortal.Tests.Scenarios/Savostin/CV_sending.feature

```diff
@@ -0,0 +1,179 @@
+	|"I agree to the personal data processing" becomes red|
+
+Scenario Outline:1.0.0_07 form with all valid data except name on it can not be sent
+	Given field name contains <name>
+	And field email contains <email>
+	And CV "Valid_file_to_attach.docx" is attached
+	And confirmation of data processing is true
+	When user clicks on "Send" button
+	Then form should not be sent
+	And following error should be displayed under field "name":
+	|"invalid name format"|
+
+	Examples: 
+	|name  |email           |
+	|Eugene|eugene@gmail.com|
+
+Scenario Outline:1.0.0_08 form with all valid data except email on it can not be sent
+	Given field name contains <name>
+	And field email contains <email>
+	And CV "Valid_file_to_attach.docx" is attached
+	And personal data processing is confirmed
+	When user clicks on "Send" button
+	Then form should not be sent
+	And following error should be displayed under field "email":
+	|"invalid email format"|
+
+	Examples: 
+	|name           |email              |
+	|Marina Sidorova|thisMailIsIncorrect|
+
+Scenario:1.0.0_09 form can not take file with invalid type
+	When user chooses file with invalid type to input
+	Then the file should not be taken
+	And the window with following text appears:
+	"""
+	Something went wrong
+	Invalid file format. Please try PDF, DOC, DOCX or RTF
+	"""
+
+Scenario:1.0.0_10 form can not take file heavier than 10MB
+	When user chooses file that is heavier than 10MB
+	Then the file should not be taken
+	And the window with following text appears:
+	"""
+	Something went wrong
+	Max file size 10 Mb 
+	"""
+
+Scenario Outline:1.0.0_11 form can not be sent without confirmation of personal data processing
+	Given field name contains <name>
+	And field email contains <email>
+	And CV "Valid_file_to_attach.docx" is attached
+	And personal data processing is not confirmed
+	When user clicks on "Send" button
+	Then form should not be sent
+	And "I agree with personal data processing" becomes red
+
+	Examples:
+	|name              |email             |
+	|Alexandr Mikhailov|testmail@gmail.com|
+
+Scenario Outline:1.0.0_12 form with all valid data and confirmation of personal data processing can be sent
+	Given field name contains <name>
+	And field email contains <email>
+	And CV "Valid_file_to_attach.docx" is attached
+	And personal data processing is confirmed
+	When user clicks on "Send" button
+	Then form should be sent
+
+	Examples:
+	|name     |email           |
+	|Test Test|testmail@mail.ru|
+
+@Bug
+Scenario Outline:1.0.0_13 forms field "mail" validates inputted data correctly
+	When user inputs <email>
+	Then the following error should be displayed under field "email"
+	"""
+	invalid email format
+	"""
+	Examples:
+	|email                            |
+	|�����������@mail.ru              |
+	|���������@mail.comcomcomcomcomcom|
+	# Fully Qualified Domain Name forbids more than 15 symbols in domain field of address.
+
+Scenario:1.0.0_14 forms field "name" can contain only 70 characters 
```

Comment:
Complicated. :)

May be, it is easier just to add the text how you want it to be added to the "Text" field?